src/assets/scripts/client/changelog/ChangelogController.js

```diff
@@ -0,0 +1,200 @@
+ *
+ * @class ChangelogController
+ */
+export default class ChangelogController {
+    /**
+     * @constructor
+     * @param {ContentQueue} contentQueue
+     */
+    constructor(contentQueue) {
+        /**
+         * The content queue, used to load in the changelog data
+         *
+         * @property contentQueue
+         * @type {ContentQueue}
+         */
+        this.contentQueue = contentQueue;
+
+        /**
+         * The DOM of the changelog container, containing version, data, and dismiss.
+         *
+         * @property $changelogContainer
+         * @type {JQuery|HTMLElement}
+         */
+        this.$changelogContainer = $(SELECTORS.DOM_SELECTORS.CHANGELOG_CONTAINER);
+
+        /**
+         * The DOM element that displays the version above the changelog
+         *
+         * @property $changelogVersion
+         * @type {JQuery|HTMLElement}
+         */
+        this.$changelogVersion = $(SELECTORS.DOM_SELECTORS.CHANGELOG_VERSION);
+
+        /**
+         * Changelog data element - where the text goes
+         *
+         * @property $changelogData
+         * @type {JQuery|HTMLElement}
+         */
+        this.$changelogData = $(SELECTORS.DOM_SELECTORS.CHANGELOG_CONTENT);
+
+        /**
+         * The link at the bottom of the container to
+         * dismiss the changelog.
+         *
+         * @property $changelogDismiss
+         * @type {JQuery|HTMLElement}
+         */
+        this.$changelogDismiss = $(SELECTORS.DOM_SELECTORS.CHANGELOG_DISMISS);
+
+        /**
+         * Toggle selector for the changelog
+         * You know, the button thing
+         *
+         * @property $changelogToggle
+         * @type {JQuery|HTMLElement}
+         */
+        this.$changelogToggle = $(SELECTORS.DOM_SELECTORS.CHANGELOG_TOGGLE);
+
+        /**
+         * A string representation of the actual changelog.
+         *
+         * @property content
+         * @type {String}
+         */
+        this.content = null;
+
+        /**
+         * The current version of openScope.
+         *
+         * @property version
+         * @type {String}
+         */
+        this.version = null;
+
+        this._setupHandlers()
+            .enable();
+    }
+
+    /**
+     * Sets up the event handlers
+     *
+     * @for ChangelogController
+     * @method _setupHandlers
+     * @private
+     * @chainable
+     */
+    _setupHandlers() {
+        this._onChangelogToggleHandler = this._onChangelogToggle.bind(this);
+
+        this.$changelogToggle.on('click', this._onChangelogToggleHandler);
+        this.$changelogDismiss.on('click', this._onChangelogToggleHandler);
+
+        return this;
+    }
+
+    /**
+     * Sets the changelog data
+     *
+     * @for ChangelogController
+     * @method enable
+     */
+    enable() {
+        this.content = '<p>Loading...</p>';
+        this.version = window.GLOBAL.VERSION;
+
+        this.$changelogVersion.text(`openScope v${this.version}`);
```

Comment:
you're doing too much work here.  I think if the basic changelog template is in an `.hbs` file already, we can leverage the build to do this work for us (see: https://github.com/openscope/openscope/blob/develop/src/templates/layout.hbs#L4)